src/Microsoft.AspNetCore.Mvc.Core/Internal/FileContentResultExecutor.cs

```diff
@@ -15,15 +17,53 @@ public FileContentResultExecutor(ILoggerFactory loggerFactory)
 
         public Task ExecuteAsync(ActionContext context, FileContentResult result)
         {
+            long rangeLength = default(long);
+
+            var range = new RangeItemHeaderValue(0, 0);
+            if (result.EnableRangeProcessing)
+            {
+                if (result.LastModified.HasValue)
+                {
+                    ComputeIfMatch(context, result, result.LastModified.Value, result.EntityTag);
```

Comment:
remove result.LastModified.Value from there, it's not being used inside the method